composer.json

```diff
@@ -1,16 +1,30 @@
 {
     "name": "krokedil/dintero-checkout-for-woocommerce",
     "description": "Dintero Checkout For WooCommerce",
+    "version": "1.0.0",
     "authors": [
         {
             "name": "krokedil",
             "email": "info@krokedil.se"
         }
     ],
     "require": {
-        "php": ">=8.0"
+        "php": ">=7.4",
+        "krokedil/shipping": "^2.0"
```

Comment:
This is not using scoping. We should probably add that here to prevent conflicts with other plugins. Since even our own package can have different versions used in them. And its better to add scoping right away to prevent having to go back and do it again later.